src/validation.cpp

```diff
@@ -1515,6 +1513,8 @@ static bool UndoReadFromDisk(CBlockUndo& blockundo, const CBlockIndex *pindex)
     return true;
 }
 
+namespace {
```

Comment:
Is this anonymous namespace still necessary? The two methods inside it are both static.